<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sayı 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sayı 2:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_sayi1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_sayi2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_toplama">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toplama</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cikarma">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Çıkarma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bolme">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bölme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_carpma">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Çarpma</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sonuc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sonuç:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
